presto-main/src/main/java/com/facebook/presto/resourcemanager/DistributedQueryInfoResource.java

```diff
@@ -137,10 +136,9 @@ public void getQueryStateInfo(@PathParam("queryId") QueryId queryId,
     private ListenableFuture<List<QueryStateInfo>> getQueryStateFromCoordinator(String xForwardedProto, UriInfo uriInfo, InternalNode coordinatorNode)
             throws IOException
     {
-        String scheme = isNullOrEmpty(xForwardedProto) ? uriInfo.getRequestUri().getScheme() : xForwardedProto;
         URI uri = uriInfo.getRequestUriBuilder()
                 .queryParam("includeLocalQueryOnly", true)
-                .scheme(scheme)
+                .scheme(coordinatorNode.getInternalUri().getScheme())
```

Comment:
Let's remove xForwardedProto and its reference?   Same for other distributed resources